web/pdf_viewer.js

```diff
@@ -45,6 +45,7 @@ var MAX_AUTO_SCALE = uiUtils.MAX_AUTO_SCALE;
 var CSS_UNITS = uiUtils.CSS_UNITS;
 var DEFAULT_SCALE = uiUtils.DEFAULT_SCALE;
 var DEFAULT_SCALE_VALUE = uiUtils.DEFAULT_SCALE_VALUE;
+var ENHANCE_TEXT_SELECTION = true;
```

Comment:
Depending on the outcome of the questions I asked in https://github.com/mozilla/pdf.js/pull/7539#issuecomment-241995058, this line may or may not stay. But if the `ENHANCE_TEXT_SELECTION` is kept, I think that it should be placed with the `DEFAULT_CACHE_SIZE` "constant" below, since placing it amongst the imported symbols makes it a bit hard to find.